src/datasources/results/constants/ResultsQueryBuilder.constants.ts

```diff
@@ -0,0 +1,183 @@
+import { QueryBuilderOperations } from 'core/query-builder.constants';
+import { QBField } from '../types/QueryResults.types';
+
+export enum ResultsQueryBuilderFieldNames {
+  PART_NUMBER = 'PartNumber',
+  PROGRAM_NAME = 'ProgramName',
+  SERIAL_NUMBER = 'SerialNumber',
+  KEYWORDS = 'Keywords',
+  PROPERTIES = 'Properties',
+  OPERATOR = 'Operator',
+  STARTED_AT = 'StartedAt',
+  UPDATED_AT = 'UpdatedAt',
+  SYSTEM_ID = 'SystemId',
+  WORKSPACE = 'Workspace',
+  STATUS = 'Status',
+  HOSTNAME = 'HostName',
+}
+
+export const ResultsQueryBuilderFields: Record<string, QBField> = {
+  PARTNUMBER: {
+    label: 'Part Number',
```

Comment:
Could we have the label similar to how we have it in Test Results filter- query builder 
![image](https://github.com/user-attachments/assets/f5a02fb7-2882-487e-80e3-7ea64a8a9f76)
```suggestion
    label: 'Part number',
```

Reply:
Please refer and make changes where ever applicable

Reply:
Updated in all places